composer.json

```diff
@@ -4,7 +4,8 @@
     "type": "library",
     "require": {
         "islandora/chullo": "^0.0.8",
-        "symfony/http-foundation": "3.2.6"
+        "symfony/http-foundation": "3.2.6",
+        "psr/log": "^1.0.1"
```

Comment:
I think if we use the monolog provider from Silex then we get automatic logging of requests and uncaught errors, though I've never tried.  Definitely not a blocker, but something to try out later maybe.